lwgenerate/_examples/aws_controltower/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lacework/go-sdk/lwgenerate/aws_controltower"
+)
+
+func controlTowerWithCloudtrail() {
+	hcl, err := aws_controltower.NewTerraform("us-east-1", "s3Arn", "Arn").Generate()
```

Comment:
I'm surprised there is no validation of inputs in the library code.  Or, maybe there is, does this code succeed?